src/backend/src/routes/design-review.routes.ts

```diff
@@ -8,4 +11,21 @@ designReviewRouter.get('/', DesignReviewController.getAllDesignReviews);
 designReviewRouter.delete('/:designReviewId/delete', DesignReviewController.deleteDesignReview);
 designReviewRouter.get('/:designReviewId', DesignReviewController.getSingleDesignReview);
 
+designReviewRouter.post(
+  '/create',
+  isDate(body('dateScheduled')),
+  nonEmptyString(body('teamTypeId')),
+  body('requiredMembers').isArray(),
+  body('optionalMembers').isArray(),
+  nonEmptyString(body('location').optional()),
+  body('isOnline').isBoolean(),
+  body('isInPerson').isBoolean(),
+  nonEmptyString(body('zoomLink').optional()),
+  body('docTemplateLink').optional(),
+  intMinZero(body('wbsElementId')),
+  body('meetingTime').isArray(),
```

Comment:
you should also check that all the elements in the meetingTime array is intMinZero